docs/development/documentation_guidelines.rst

```diff
@@ -25,6 +25,8 @@ To get started editing the docs:
 
       pip install --no-deps --require-hashes -r securedrop/requirements/python3/develop-requirements.txt
 
+.. note:: Ensure you are running in a virtual environment, so you can install, upgrade, and remove packages using the program `pip`. Run the command `python3 -m venv .venv` to create and manage your virtual environments.
```

Comment:
This is definitely a practice to encourage, so I think you could explicitly make it part of the  docs setup process rather than relegating it to a note. Elsewhere in the [docs](https://docs.securedrop.org/en/stable/development/setup_development.html#setting-up-a-multi-machine-environment) we recommend using `virtualenvwrapper`, so it might make sense to have this be consistent with those instructions. Leveraging includes could make sense here.

Reply:
I think the doc flow should be reorganized, with a section that calls out setting up the VM. I included the note as a place holder until that re-architecture is finalized. Do you think this note is okay for the time being, or should I hold off until we restructure the flow?

Reply:
You can follow the examples in `docs/includes` (`grep` through the docs to see how these are used) to create a re-usable text block `virtualenv.txt` which we can initially insert at this position and then place in other relevant sections of the text. 

Per discussion at standup, I suggest we go with `python3 -m venv .venv` as the canonical method (since it will always work without installing additional dependencies) and mention the existence of other convenience tools like `virtualenvwrapper`, which provides the `mkvirtualenv` shell function.

Reply:
Also, as just discussed, let's add instructions for how to activate a virtualenv.